src/components/Waveform.js

```diff
@@ -1,9 +1,89 @@
-import React from 'react';
+import React, { Component } from "react";
+import Peaks from "peaks.js";
+import { Button, ButtonToolbar, FormControl, Form } from "react-bootstrap";
 
-const Waveform = () => (
-  <section className="waveform-section">
-    <div className="waveform-holder" />
-  </section>
-);
+import soundMP3 from "../data/TOL_6min_720p_download.mp3";
+import soundOGG from "../data/TOL_6min_720p_download.ogg";
+import soundJSON from "../data/TOL_6min_720p_download.json";
+import soundDAT from "../data/TOL_6min_720p_download.dat";
+
+class Waveform extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      peaksInstance: null,
+      seekTime: ""
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentDidMount() {
+    const options = {
+      container: this.refs.waveformContainer,
+      mediaElement: this.refs.mediaPlayer,
+      dataUri: {
+        arraybuffer: soundDAT,
+        json: soundJSON
+      },
+      keyboard: true,
+      pointMarkerColor: "#006eb0",
+      showPlayheadTime: true
+    };
```

Comment:
Typically we'll want to keep config object options, separated or independent as possible.  I'm working on some updates now, playing around with some ideas, and will push up, then maybe we can discuss?